tests/combo/converter/combo_atomeseUTest.cxxtest

```diff
@@ -45,10 +45,10 @@ public:
 		combo::ComboToAtomese to_atomese;
 		Handle h = to_atomese(tr);
 
-		HandleSeq list_or = {createNode(PREDICATE_NODE, "1"), createNode(PREDICATE_NODE, "2")};
+		HandleSeq list_or = {createNode(PREDICATE_NODE, "$1"), createNode(PREDICATE_NODE, "$2")};
```

Comment:
@kasimebrahim , `$` has been traditionally used for `VARIABLE_NODE`. Not saying it is forbidden for other node types but I wonder if this could lead to confusions. What do you think?

Reply:
yeah it is, what I had in mind was id::argument is also traditionally prepended with `$` and adding and removing `$` when converting to and from combo might be more confusing.

Reply:
> id::argument is also traditionally prepended 

not just traditionally

Reply:
I understand. I don't mind keeping `$` for now, we can decide after time whether it is more confusing or enlightening.